core/storage/job/gae_models.py

```diff
@@ -89,6 +89,12 @@ def get_new_id(cls, entity_name):
 
     @property
     def is_cancelable(self):
+        """Checks if the job is cancelable.
+
+        Returns:
+            bool. True if the status_code is queued or started, otherwise
+            false.
```

Comment:
Indent the continuation line no.-- 96 by 4 spaces.

Reply:
Hey @vibhor98 , I've done that now. Am I required to submit a new PR?